Content.Shared/Doors/Systems/SharedDoorSystem.cs

```diff
@@ -577,18 +598,21 @@ public IEnumerable<EntityUid> GetColliding(EntityUid uid, PhysicsComponent? phys
 
             //TODO: Make only shutters ignore these objects upon colliding instead of all airlocks
             // Excludes Glasslayer for windows, GlassAirlockLayer for windoors, TableLayer for tables
-            if (otherPhysics.Comp.CollisionLayer == (int) CollisionGroup.GlassLayer || otherPhysics.Comp.CollisionLayer == (int) CollisionGroup.GlassAirlockLayer || otherPhysics.Comp.CollisionLayer == (int) CollisionGroup.TableLayer)
+            if (otherPhysics.Comp.CollisionLayer == (int)CollisionGroup.GlassLayer ||
+                otherPhysics.Comp.CollisionLayer == (int)CollisionGroup.GlassAirlockLayer ||
+                otherPhysics.Comp.CollisionLayer == (int)CollisionGroup.TableLayer)
```

Comment:
```suggestion
            if (otherPhysics.Comp.CollisionLayer == (int)CollisionGroup.GlassLayer
                || otherPhysics.Comp.CollisionLayer == (int)CollisionGroup.GlassAirlockLayer
                || otherPhysics.Comp.CollisionLayer == (int)CollisionGroup.TableLayer)
```
other multi-line expressions put the operator at the start of lines.